configs/nri-prometheus-config.yml.sample

```diff
@@ -0,0 +1,27 @@
+integrations:
+  - name: nri-prometheus
+    inventory_source: integration/com.newrelic.prometheus
```

Comment:
I think we could remove this and use the default instead. `inventory_source` is set by default as integration/nri-prometheus . [ docs ](https://docs.newrelic.com/docs/create-integrations/infrastructure-integrations-sdk/specifications/host-integrations-newer-configuration-format#inventory_source).

Reply:
Do you know why it was set explicitly here, @gsanchezgavier? Removing it here and using the default will change the actual name.

Reply:
no i don't, but inventory is not being used afaik here.

Reply:
I'm just keeping it like this for now, and discuss this with Paolo when he's back. We'll not do a release with it yet, anyway.